sktime/forecasting/base/_base.py

```diff
@@ -210,8 +213,7 @@ def predict(
             Prediction intervals
         """
         # handle inputs
-        self.check_is_fitted()
-        self._set_fh(fh)
+        self.perform_checks(to_be_checked={"is_fitted", "fh"}, fh=fh)
```

Comment:
Do we need to wrap this into a method? I think I'd prefer to just see the lines of code here, unless perhaps this is re-used somewhere else?